CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Dates are in `yyyy-mm-dd`.
 
 ### Fixed
 
+## Version 1.8.6 (verCode 1080600), 2021-06-16
+### Changed
+* ProgressDialog in TokenActivity with on screen prompts
+
```

Comment:
You don't need to mark a new version in this commit. We'll do that when we are ready for a release.
Instead just add an entry under Unreleased->Changed. When we are ready for a release, we'll change the Unreleased heading to the new version number.
https://github.com/crux-bphc/CMS-Android/commit/545d607c1c7457c543121fb980f1a862a416c864
Check this commit (and the immediately next child commit on the` development` branch).